README.md

```diff
@@ -204,7 +213,7 @@ print("Name: ", entity)
 # Each entity is linked to UMLS with a score
 # (currently just char-3gram matching).
 for umls_ent in entity._.kb_ents:
-	print(linker.kb.cui_to_entity[umls_ent[0]])
+	print(nlp.get_pipe("scispacy_linker").kb.cui_to_entity[umls_ent[0]])
```

Comment:
Nit: can you pull this `get_pipe` out of the loop, it makes it clearer to read

Reply:
sure, will do